src/components/shared/transactionSummary/footer.js

```diff
@@ -57,7 +61,10 @@ const SecondPassInput = ({
     <div className={styles.secondPassphrase}>
       <PassphraseInput
         t={t}
-        onFill={set2ndPass}
+        onFill={(passPhrase, validationError) => {
+          setIsValid(validationError);
+          set2ndPass(passPhrase);
+        }}
```

Comment:
Please replace the state hook with a custom hook that validates the second passphrase and stored the `secondPassphrase` as:

```
{
   data: '',
   error: 0,
   feedback: [],
}
```

Where error is `-1` for empty, `0` for valid, and a positive number for the number of errors discovered.
And feedback is an array of strings, each representing an error message.